VideoWeb/VideoWeb.EventHub/Hub/IEventHubClient.cs

```diff
@@ -12,7 +12,7 @@ public interface IEventHubClient
         Task ConsultationMessage(Guid conferenceId, string requestedBy, string requestedFor, string result);
         Task AdminConsultationMessage(Guid conferenceId, RoomType room, string requestedFor, ConsultationAnswer? answer = null);
         Task HelpMessage(Guid conferenceId, string participantName);
-        Task ReceiveMessage(Guid conferenceId, string from, string message, DateTime timestamp, Guid messageUuid);
+        Task ReceiveMessage(Guid conferenceId, string from, string message, DateTime timestamp, Guid messageUuid, bool isJudge);
```

Comment:
No neeed to extend the model here, not a suitable solution for when participant IM is enabled.

Reply:
done